src/ansys/aedt/core/visualization/report/common.py

```diff
@@ -641,7 +641,7 @@ def traces(self):
         except Exception:
             return _traces
         for el in oo_names:
-            if "Families" not in oo.GetChildObject(el).GetPropNames():
+            if not {"Families", "Source"}.intersection(set(oo.GetChildObject(el).GetPropNames())):
```

Comment:
I like this change. I would have used something like `not any(map(lambda prop: prop in oo.GetChildObject(el).GetPropNames, ("Families", "Source")))` but what you propose is more clear and avoid multiple calls to the API ! Since we are working with sets, what do you think about the following change ?
```suggestion
            if {"Families", "Source"}.isdisjoint(set(oo.GetChildObject(el).GetPropNames())):
```